test/test_common.h

```diff
@@ -55,10 +55,11 @@ static inline int compare_tm(struct tm *t1, struct tm *t2)
 static inline int verbose_test(void)
```

Comment:
Don't do getenv at each check. Do it once in initialization and cache this value, which you shall return from verbose_test.